### PR TITLE
Add self-update URL

### PR DESCRIPTION
**Pull request by imobach**
Add self_update_url element. Requires https://github.com/yast/yast-installation-control/pull/20.

### CODE REVIEW

File: control/control.SLES.xml [hunk 1]
```diff
@@ -115,6 +115,9 @@ textdomain="control"
                 <packages>sblim-sfcb</packages>
             </service>
         </services_proposal>
+
+        <!-- FATE: #319716: Automatic Update of YaST at beginning of the installation -->
+        <!-- <self_update_url>http://update.opensuse.org/update.dud</self_update_url> -->
```
Comment (mvidner) [inline control/control.SLES.xml]: Are you sure you want it commented out?

Comment (imobach) [inline control/control.SLES.xml]: Yes, because we don't have a definitive URL yet and I prefer a comment instead using a fake URL as the PBI suggested.

Comment (mvidner) [inline control/control.SLES.xml]: It does not make sense. The changelog says something, but the control file does not implement it. Either define *some* URL, or don't submit anything.

**Comment by imobach** [comment]
@mvidner Well, you're right. Then I'll hold this PR for now. Thanks!

File: control/control.SLES.xml (hunk 1 above)
Comment (imobach) [inline control/control.SLES.xml]: @mvidner Well, you're right. Then I'll hold this PR for now. Thanks!

**Pull request closed without merging**